offerzen/global/employers/prospects@2.2.js

```diff
@@ -8,50 +8,44 @@ window.$loaded(function () {
       $parsleyLoaded(cb);
     }, 50);
   };
+
   window.$parsleyLoaded(function (window, document, parsley) {
+    const emailOnlyProspectForm = $('#Company-Lead-Form-Email-Only');
+
     function retrieveProspectJourney() {
-      if ('emailValue' in localStorage && 'acceptedTerms' in localStorage) {
+      if ('emailValueC' in localStorage) {
```

Comment:
I see you're using it like this lower down as well. Is this for "variant C"?

Reply:
Yeah this is for Variant C. I didn't want to make it obvious by stating variantC for users. 
It's important to keep the two emailValues separate depending on which experiment the user previously started so that there are no conflicts on the forms for people with ad blockers and the correct step is retrieved properly for each user journey.
@jethroflanagan  what do you suggest?

Reply:
I'll add a comment to calrify